cubeseed_login/src/pages/moderator/users/accepted_users/AcceptedUserCard/[card].tsx

```diff
@@ -0,0 +1,93 @@
+import React from "react"
+import "@/styles/AcceptedUsersCard.css"
+import "tailwindcss/tailwind.css"
+import {
+  CheckmarkCircleOutline,
+  CloseCircleOutline,
+  FolderOutline,
+} from "react-ionicons"
+import SideBar from "@/component/moderator_sidebar/sidebar"
+import { useRouter } from "next/router"
+
+interface Farmer {
+  id: number
+  name: string
+  email: string
+  time: string
+  status: string
+  certificate: string | null
+  code: number | null
+  document1: string | null
+  document2: string | null
+}
+
```

Comment:
It's a good practice to centralize reusable types/interfaces in a shared location rather than duplicating them across multiple components so it can be easily maintained. You can create a separate file ( types.ts) in the `lib` folder and export the `Farmer interface` from there.

```
types.ts
export interface Farmer {
  id: number;
  name: string;
  email: string;
  time: string;
  status: string;
  certificate: string | null;
  code: number | null;
  document1: string | null;
  document2: string | null;
}
```